server/test/bin/lfortran

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+cat "${@:$#}"
```

Comment:
do we need this file?

Reply:
Yes, I use this script in the unit tests to avoid adding a dependency on lfortran (it lets us mock responses by echoing file contents). We need to add end-to-end tests that include lfortran as a dependency but those will be slow to initialize and too heavyweight for unit testing.